### PR TITLE
Add sig-docs to the contribution map

### PR DESCRIPTION
**Pull request by qiancai**
Add the row for sig-docs to the table in the SIG - Special Interest Group section

### CODE REVIEW

File: maps/contribution-map.md [hunk 1]
```diff
@@ -183,6 +185,7 @@ Contribution tutorial:
 | *SIG Name* | *Description* | *Join In* |
 | ---- | ---- | ---- |
 | DDL(TiDB) | Covers the TiDB DDL and parser parts. Work includes supporting new features or improving the syntax compatibility between TiDB and MySQL, fixing all kinds of DDL-related and parser-related bugs, and etc. [Readme](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl) | channel: [sig-ddl](https://slack.tidb.io/invite?team=tidb-community&channel=sig-ddl&ref=github-sig) |
+| Docs | Covers the documentation of all PingCAP products, tools, or projects, both [in English](https://pingcap.com/docs/stable/) and [in Chinese](https://pingcap.com/docs-cn/stable/).  [Readme](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs) | channel: [sig-docs](https://slack.tidb.io/invite?team=tidb-community&channel=sig-docs&ref=github-sig) |
```
Comment (lilin90) [inline maps/contribution-map.md:188]: Let's avoid using the outdated links, though they will redirect to the new links.
```suggestion
| Docs | Covers the documentation of all PingCAP products, tools, or projects, both [in English](https://docs.pingcap.com/tidb/stable) and [in Chinese](https://docs.pingcap.com/zh/tidb/stable).  [Readme](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs) | channel: [sig-docs](https://slack.tidb.io/invite?team=tidb-community&channel=sig-docs&ref=github-sig) |
```

File: maps/contribution-map.md [hunk 2]
```diff
@@ -4,22 +4,24 @@
...
+  - [Ecosystem Tools - Lightning: A high-speed data import tool for TiDB](#ecosystem-tools---lightning-a-high-speed-data-import-tool-for-tidb)
+  - [Ecosystem Tools - BR: A command-line tool for distributed backup and restoration of the TiDB cluster data](#ecosystem-tools---br-a-command-line-tool-for-distributed-backup-and-restoration-of-the-tidb-cluster-data)
+  - [TiDB on K8S/Docker : Creates and manages TiDB clusters running in Kubernetes](#tidb-on-k8sdocker--creates-and-manages-tidb-clusters-running-in-kubernetes)
+  - [Deployment Tools - tidb-ansible: A tool to capture data change of TiDB cluster](#deployment-tools---tidb-ansible-a-tool-to-capture-data-change-of-tidb-cluster)
+  - [Chaos-Mesh: A Chaos Engineering Platform for Kubernetes](#chaos-mesh-a-chaos-engineering-platform-for-kubernetes)
+  - [Documentation](#documentation)
+  - [TUG(CN)](#tugcn)
+  - [PingCAP University(CN)](#pingcap-universitycn)
+  - [SIG - Special Interest Group](#sig---special-interest-group)
```
Comment (lilin90) [inline maps/contribution-map.md:24]: It seems that updates for this section are unnecessary.

File: maps/contribution-map.md (hunk 1 above)
Comment (qiancai) [reply maps/contribution-map.md]: Thanks. I've updated the outdated links to the new links you provided. PTAL.

File: maps/contribution-map.md (hunk 2 above)
Comment (qiancai) [reply maps/contribution-map.md]: Yes. I've reverted the changes. PTAL.

File: maps/contribution-map.md [hunk 3]
```diff
@@ -183,6 +183,7 @@ Contribution tutorial:
 | *SIG Name* | *Description* | *Join In* |
 | ---- | ---- | ---- |
 | DDL(TiDB) | Covers the TiDB DDL and parser parts. Work includes supporting new features or improving the syntax compatibility between TiDB and MySQL, fixing all kinds of DDL-related and parser-related bugs, and etc. [Readme](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl) | channel: [sig-ddl](https://slack.tidb.io/invite?team=tidb-community&channel=sig-ddl&ref=github-sig) |
+| Docs | Covers the documentation of all PingCAP products, tools, or projects, both [in English](https://docs.pingcap.com/tidb/stable) and [in Chinese](https://docs.pingcap.com/zh/tidb/stable).  [Readme](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs) | channel: [sig-docs](https://slack.tidb.io/invite?team=tidb-community&channel=sig-docs&ref=github-sig) |
```
Comment (lilin90) [inline maps/contribution-map.md:186]:
```suggestion
| Docs | Covers the documentation of all PingCAP products, tools, or projects, both [in English](https://docs.pingcap.com/tidb/stable) and [in Chinese](https://docs.pingcap.com/zh/tidb/stable). [Readme](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs) | channel: [sig-docs](https://slack.tidb.io/invite?team=tidb-community&channel=sig-docs&ref=github-sig) |
```

**Comment by tisonkun** [comment]
The map structure has been changed. PTAL @qiancai

**Comment by qiancai** [comment]
> The map structure has been changed. PTAL @qiancai

Hi @tisonkun, thanks for the information. I've checked the latest map document and added a row for Documentation Slack Channel to the TiDB Developer Group table. PTAL.

File: maps/contribution-map.md [hunk 4]
```diff
@@ -182,3 +182,4 @@ Contribution tutorial:
...
 | [TiDB Slack Channel](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) | The official Slack channel of the TiDB community.                                                                                   |
 | [TiKV Slack Channel](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzWYZWYZWYWJGWZGWYG)                                                     | The official Slack channel of the TiKV community.                                                                                   |
+| [Documentation Slack Channel](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs)                                                     | The official Slack channel of the PingCAP Documentation community.                                                                                   |
```
Comment (tisonkun) [inline maps/contribution-map.md:185]:
```suggestion
| [Documentation Slack Channel](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs)                                                     | The official Slack channel of the TiDB Documentation community.                                                                                   |
```

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
Thanks! Merging...

**Pull request merged**